src/redux/api/index.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+
+export const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/'; 
+export const appId = 'bJTtiRJfwS5VIiVRUXvf';
+
+export const getBooksFromApi = async () => {
+  const finalData = [];
+  const response = await axios.get(`${baseUrl}${appId}/books`);
+  // eslint-disable-next-line array-callback-return
```

Comment:
- [ ] Kindly remove this eslint disable line its best to fix the issue in the code.

Reply:
Hello, thanks for the review. I have removed it and other other eslint disable line in the other files